test/Libraries/Microsoft.Extensions.Diagnostics.Probes.Tests/KubernetesProbesOptionsValidatorTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace Microsoft.Extensions.Diagnostics.Probes.Test;
+
+public class KubernetesProbesOptionsValidatorTests
```

Comment:
Can we also add another test that KubernetesProbesOptions are validated when we use AddKubernetesProbes method?

Reply:
Hi @mobratil, I've updated the unit tests, please check `Validator_WhenHostStarts_Fails` and `Validator_WhenHostStarts_Succeeds`? Thank you!